### PR TITLE
Math_magicians: components

### PR DESCRIPTION
**Pull request by elisha2kyakpo1**
In this branch:

1. I removed all the boilerplate code/files.
2. I added Calculator.js inside a component folder
3. Add calculator.css for styling.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import './Calculator.css';
+
+export default class Calculator extends React.Component {
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [ ] Please use ES6 syntax.  It allows you to write less code and do more.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,13 @@
...
-    </div>
-  );
+export default class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <Calculator />
+      </div>
+    );
+  }
```
Comment (arslanbisharat) [inline src/App.js]: - [ ] The App.js component should be class-based. Please change this function to class-based.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 2 above)
Comment (elisha2kyakpo1) [reply src/App.js]: If you check properly, you will see that App.js is a class-based component

File: src/App.js [hunk 3]
```diff
@@ -1,27 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
```
Comment (arslanbisharat) [inline src/App.js]: - [ ] Please make sure you are not disabling eslint rules. These are here to help you write clean code.

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [ ]  Please make sure you are not disabling eslint rules. These are here to help you write clean code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 2 above)
Comment (arslanbisharat) [reply src/App.js]: I am sorry I mistakenly asked for this change.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**